### ISSUE TITLE
Support for DGraph

### ISSUE THREAD

**Issue by dwimberger** [apollostack/apollo#32]
Recently stumbled over DGraph. Seems to become an interesting match for the "application server" (or at least the lower layers) as it natively supports GraphQL.

http://dgraph.io/
https://github.com/dgraph-io/dgraph

I am not familiar enough with the Apollo Stack yet, but would you think it matches and that it could be done?

**Comment by batjko** [comment]
If DGraph exposes a native GraphQL API, querying it with an Apollo client should be just fine, I'm guessing.
It seems to market itself as a big data graph database, so Apollo's reactivity would have to be able to handle large floods of data sets.

Either way, they're still in an Alpha stage themselves, just about to release v0.3, with incomplete GraphQL support (at least for v0.1 according to their [Github README](https://github.com/dgraph-io/dgraph#current-status))

**Comment by helfer** [comment]
Yes, Apollo (and GraphQL for that matter) will work with any backend. The fact that DGraph supports GraphQL doesn't actually matter all that much, other than making things a little bit easier.

From a cursory glance, I have the impression that what they call GraphQL isn't actually spec-compliant, at least not for mutations. I don't know if that's intentional though.
